profiles/common/modules/features/nexteuropa_trackedchanges/nexteuropa_trackedchanges.module

```diff
@@ -0,0 +1,747 @@
+    $form['#submit'][$feature_set_submit_key] = '_nexteuropa_trackedchanges_disable_check_submit';
+  }
+}
+
+/**
+ * Implements hook_field_widget_WIDGET_TYPE_form_alter().
+ *
+ * It implements it for "Text area" widget.
+ */
+function nexteuropa_trackedchanges_field_widget_text_textarea_form_alter(&$element, &$form_state, $context) {
+  _nexteuropa_trackedchanges_field_widget_tracking_form_alter($element, $form_state, $context);
+}
+
+/**
+ * Implements hook_field_widget_WIDGET_TYPE_form_alter().
+ *
+ * It implements it for "Text area with a summary" widget.
+ */
+function nexteuropa_trackedchanges_field_widget_text_textarea_with_summary_form_alter(&$element, &$form_state, $context) {
+  _nexteuropa_trackedchanges_field_widget_tracking_form_alter($element, $form_state, $context);
+}
+
+/**
+ * Implements hook_field_widget_WIDGET_TYPE_form_alter().
+ *
+ * It implements it for "Text field" widget.
+ */
+function nexteuropa_trackedchanges_field_widget_text_textfield_form_alter(&$element, &$form_state, $context) {
+  _nexteuropa_trackedchanges_field_widget_tracking_form_alter($element, $form_state, $context);
+}
+
+/**
+ * Implements hook_field_attach_form().
+ */
+function nexteuropa_trackedchanges_field_attach_form($entity_type, $entity, &$form, &$form_state, $langcode) {
+  // Only node entities are supported.
+  $tracking_settings = _nexteuropa_trackedchanges_wysiwyg_tracking_settings($entity);
+  if (($entity_type == 'node') && $tracking_settings['is_activate']) {
+    if (!isset($form['language'])) {
+      // The form item containing the entity can change from an entity to
+      // another. Then we set it like that to easy future treatment.
+      $form['language'] = array(
+        '#type' => 'value',
+        '#value' => $langcode,
+      );
+    }
+    $form['#validate'][] = '_nexteuropa_trackedchanges_tracking_entity_validate';
+    $form['actions']['submit']['#submit'][] = '_nexteuropa_trackedchanges_tracking_entity_log';
+  }
+}
+
+/**
+ * Implements hook_entity_view_alter().
+ */
+function nexteuropa_trackedchanges_entity_view_alter(&$build, $type) {
+  _nexteuropa_trackedchanges_ckeditor_lite_view_alter($build, $type);
+}
+
+/**
+ * Implements hook_tmgmt_dgt_html_filter_formats_containing_html().
+ */
+function nexteuropa_trackedchanges_tmgmt_dgt_html_filter_formats_containing_html() {
+  return array(
+    'full_html_track',
+  );
+}
+
+/**
+ * Implements hook_wysiwyg_editor_settings_alter().
+ *
+ * Alter the display and the status (active/inactive) of CKEditor LITE buttons.
+ */
+function nexteuropa_trackedchanges_wysiwyg_editor_settings_alter(&$settings, $context) {
+  $state = multisite_wysiwyg_set_current_entity_object_state();
+
+  if ($context['profile']->editor == 'ckeditor') {
+    // If ckeditor_lite_istracking is not set, the default value is 1.
+    // For feature needs, the logic is inverted.
+    $settings['lite']['isTracking'] = variable_get('ckeditor_lite_istracking') ? TRUE : FALSE;
```

Comment:
Why not use the second parameter of variable_get() like:

`$settings['lite']['isTracking'] = (bool) variable_get('ckeditor_lite_istracking', FALSE);`